python/cuml/test/conftest.py

```diff
@@ -1,20 +1,53 @@
-import os
+#
+# Copyright (c) 2018-2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 
+import os
+import sys
 import cupy as cp
 import cupyx
 import pytest
 from pytest import Item
 from sklearn.datasets import fetch_20newsgroups
 from sklearn.feature_extraction.text import CountVectorizer
-from cuml.dask.preprocessing.LabelEncoder import LabelEncoder as dask_label
-from cuml.preprocessing.LabelEncoder import LabelEncoder
 import numbers
 
+
 # Stores incorrect uses of CumlArray on cuml.common.base.Base to print at the
 # end
 bad_cuml_array_loc = set()
 
 
+def checked_isinstance(obj, class_path_list):
+    """
+    Small helper function to check instance of object that doesn't import
+    class_path at import time, only at check time. Returns False if
+    class_path cannot be imported.
+    """
+    ret = False
+    for class_path in class_path_list:
+        module_name, class_name = class_path.rsplit(".", 1)
+        module = sys.modules[module_name]
+        module_class = getattr(module, class_name, None)
```

Comment:
Wish there were a simpler way to do this, but I wasn't able to come up with one...

Reply:
you'd think there would be a built in function in python to check for module existence in a safe manner, no?